src/Libpng-2.0.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="Libpng-2.0" name="libpng License 2.0">
+      <crossRefs>
+         <crossRef>http://www.libpng.org/pub/png/src/libpng-LICENSE.txt</crossRef>
+      </crossRefs>
+
+    <text>
+      <p>PNG Reference Library License version 2</p>
+      <alt match="^[-]*$" name="dashed-line">---------------------------------------</alt>
+      <list>
```

Comment:
Should this list be within a `<copyrightText>` block, so that the copyright notices are tagged as such and are optional?

Reply:
Good point! I think the title should be in a `<titleText>` block, with the dashed line optional. I think the line is made "optional" by the regex in the match parameter, but should it also be `<optional>` or in `<copyrightText>`?

Reply:
I would suggest enclosing the list of copyrights in a `<copyrightText>` block.  I also agree wrapping the title would be a good idea - this would allow the matching algorithm to treat the title as optional.